australian-guide-to-legal-citation.csl

```diff
@@ -195,11 +195,11 @@
   </macro>
   <macro name="title-short">
     <choose>
-      <if type="book legislation webpage thesis motion_picture manuscript" match="any">
-        <text variable="title" font-style="italic" text-case="title" form="short"/>
+      <if type="book legislation webpage thesis motion_picture manuscript legal_case article" match="any">
+        <text variable="title-short" font-style="italic" text-case="title" form="short"/>
       </if>
       <else>
-        <text variable="title" quotes="true" text-case="title" form="short"/>
+        <text variable="title" quotes="false" text-case="title" form="short"/>
```

Comment:
As above. I'm not as careful checking my work as I should be. Sorry. 

I know it wastes your time if I don't. I'm not sure how I missed so much.